docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -1,26 +1,30 @@
 # Pipelines
 
-We previously introduced [Nodes](./nodes.md) as building blocks that represent tasks, and can be combined in a pipeline to build your workflow. A pipeline organises the dependencies and execution order of your collection of nodes, and connects inputs and outputs while keeping your code modular. The pipeline resolves dependencies to determine the node execution order, and does *not* necessarily run the nodes in the order in which they are passed in.
+A pipeline is a set of [Nodes](./nodes.md) that will be automatically resolved after pipeline creation to determine the node execution order based on outputs and inputs of each node.
 
-To benefit from Kedro's automatic dependency resolution, you can chain your nodes into a {py:class}`~kedro.pipeline.Pipeline`, which is a list of nodes that use a shared set of variables.
+Technically {py:class}`~kedro.pipeline.Pipeline` is a python class that could be created using {py:class}`~kedro.pipeline.modular_pipeline.pipeline` method, based on Nodes or other Pipelines(in that case all nodes from that pipeline will be used).
 
-## How to build a pipeline
+Most important about kedro pipelines:
+- [How to create a simple pipeline](#how-to-create-a-simple-pipeline)
+- [Create a new blank pipeline with kedro pipeline create command](#create-a-new-blank-pipeline-with-the-kedro-pipeline-create-command)
+- [Pipeline creation structure](#pipeline-creation-structure)
+- [Key Pipeline methods and operations](#key-pipeline-methods-and-operations)
+- [Bad pipelines - rules](#bad-pipelines)
+- [Custom new pipeline templates](#custom-new-pipeline-templates)
```

Comment:
I think these headings could be all "how to" headlines like we have in configuration docs. https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#advanced-configuration-for-kedro-projects